code/src/java/plugin/lsttokens/NaturalattacksLst.java

```diff
@@ -122,186 +122,160 @@ protected ParseResult parseTokenWithSeparator(LoadContext context,
 			{
 				return pr;
 			}
-			Equipment anEquip = createNaturalWeapon(context, obj, tokString.intern());
+			String wpn = tokString.intern();
 
-			if (anEquip == null)
+			StringTokenizer commaTok = new StringTokenizer(wpn, Constants.COMMA);
+
+			int numTokens = commaTok.countTokens();
+			if (numTokens < 4)
```

Comment:
extract Magic Number

Reply:
Might be useful to know for the developer what `4` means - ILLEGAL_NUMBER_OF_TOKENS?  Something else?